fastly/kinesis_test.go

```diff
@@ -0,0 +1,280 @@
+package fastly
+
+import "testing"
+
+func TestClient_Kineses(t *testing.T) {
+	t.Parallel()
+
+	var err error
+	var tv *Version
```

Comment:
**Nitpick** (apologies, no really, this is such a nitpick but my OCD is eating me alive right now).

Can we please rename `tv` to just `v` so it more closely aligns with the golang convention of naming a variable after the type assigned to the variable.

Otherwise later in the code when seeing `Version: tv.Number` it looks a bit ambiguous to me what `tv` means. 

So, for example, I had to scroll back up to see it's named that way because of the `testVersion()` function used, but naming it after the function isn't the same thing as naming it after the actual 'type' that's assigned to the variable.

Reply:
Done